lib/status.js

```diff
@@ -81,12 +86,45 @@ class Status extends EventEmitter {
    * Get the cumulative stats for the current window
    */
   get stats () {
-    return this[WINDOW].reduce((acc, val) => {
+    const totals = this[WINDOW].reduce((acc, val) => {
       // the window starts with all but one bucket undefined
       if (!val) return acc;
-      Object.keys(acc).forEach(key => (acc[key] += val[key] || 0));
+      Object.keys(acc).forEach(key => {
+        if (key !== 'latencyTimes' && key !== 'percentiles') {
+          (acc[key] += val[key] || 0);
+        }
+      });
+
+      if (this.rollingPercentilesEnabled) {
+        acc.latencyTimes.push.apply(acc.latencyTimes, val.latencyTimes || []);
```

Comment:
we need to concatenate the values in that `latencyTimes` so if we just did push, then we would get something like this: `[ [1], [2], [3], [4] ]` instead of `[1,2,3,4]` .   would be nice if we could use the spread operator here, but i don't think it goes back to node 4